views/js/runner/plugins/security/fullScreen.js

```diff
@@ -251,31 +269,52 @@ define([
             }
 
             function alertUser() {
-                if (!waitingForUser) {
-                    if (fullScreenSupported) {
-                        waitingForUser = true;
-                        stopFullScreenChangeObserver();
-                        disableItem();
-                        testRunner.trigger('alert.fullscreen', message, function(reason) {
-
-                            if (reason === 'esc') {
-                                waitingForUser = false;
-                                return alertUser();
-                            }
+                if (!waitingForUser && fullScreenSupported) {
+                    waitingForUser = true;
+                    stopFullScreenChangeObserver();
+                    disableItem();
+
+                    if(isIE11()) {
+                        testRunner.trigger('alert.fullscreen', messageIE11, userAlertCallbackIE11, dialogParams);
+                    } else {
+                        testRunner.trigger('alert.fullscreen', message, userAlerCallback, dialogParams);
+                    }
 
-                            requestFullScreen();
+                }
+            }
+            function userAlertCallbackIE11(reason) {
+                if (reason === 'esc') {
+                    waitingForUser = false;
+                    return alertUser();
+                }
 
-                            _.defer(function() {
-                                waitingForUser = false;
-                                enableItem();
+                _.defer(function() {
+                    waitingForUser = false;
 
-                                if (!fullScreenSupported) {
-                                    startFullScreenChangeObserver();
-                                }
-                            });
-                        }, dialogParams);
+                    if(checkFullScreen()) {
+                        enableItem();
+                    } else {
+                        alertUser();
                     }
+                });
+            }
+
+            function userAlerCallback(reason) {
```

Comment:
nitpick: may be `userAlertCallback` instead if `userAlerCallback`
```suggestion
            function userAlertCallback(reason) {
```